helm/alfresco-content-services/templates/config-repository.yaml

```diff
@@ -154,3 +154,4 @@ data:
       -Dmessaging.broker.url=$BROKER_URL
       -Dmessaging.broker.username=$BROKER_USERNAME
       -Dmessaging.broker.password=$BROKER_PASSWORD
+      -Dencryption.ssl.truststore.location=/etc/pki/java/cacerts
```

Comment:
still wondering why it's needed given that `/usr/lib/jvm/java/lib/security/cacerts` is already a symlink to `/etc/pki/java/cacerts`

Reply:
becuase this `encryption.ssl.truststore.*` are actually linked to an Alfresco implementation of keystore which doesn't default to the JVM keystores:
https://github.com/Alfresco/alfresco-enterprise-repo/blob/master/repository/src/main/java/org/alfresco/repo/search/impl/elasticsearch/client/ElasticsearchHttpClientFactory.java#L83

Reply:
Other than that, we should give a path relative to JAVA_HOME so it doesn't break if we change JDK distribution or OS